css/css-ui/compute-kind-widget-generated/WEB_FEATURES.yml

```diff
@@ -0,0 +1,4 @@
+features:
+- name: background-attachment
+  files:
+  - kind-of-widget-fallback-*-background-attachment-*
```

Comment:
These tests seem to be for https://drafts.csswg.org/css-ui-4/#appearance-disabling-properties which I'd probably classify as tests for the `appearance` property. But I suggest just skipping them in this PR.